app/src/main/report/services/ReportsService.ts

```diff
@@ -8,8 +8,7 @@ export class ReportsService extends AbstractLocalService {
     }
 
     getReportVersions(reportId: string) {
-        return this.setOptions({cacheKey: ReportsCacheKeysEnum.versions})
```

Comment:
Hmm, maybe `ignoreCache` should be an argument that the method takes, so that we can still use the cache in other contexts?

Reply:
The trouble is that the only other place where we call getReportVersions (on page load) we need to ignore the cache too to ensure we pick up freshly run versions (I tried putting in this param for the refresh from RunReport component only, and found my new version disappeared when I navigated to it!). I suppose we could put in some logic to say you can safely ignore the cache if user does not have run perms, or there have been no running reports. Or is the issue that when we set options to not use the cache for fetching, we might still want to update the cache with what we fetch?

Reply:
Ah ok, leave this then. It's not a lot of data to request each time.